packages/cli/src/lib.rs

```diff
@@ -241,11 +241,10 @@ fn to_valid_pkg_name(project_name: &str) -> String {
         .replace('\\', "")
         .replace('/', "");
 
-    if let Some(ch) = ret.chars().next() {
-        if ch.is_ascii_digit() || ch == '-' {
-            ret.remove(0);
-        }
-    }
+    let ret = ret
+        .chars()
+        .skip_while(|ch| ch.is_ascii_digit() || *ch == '-')
```

Comment:
this will still remove all numbers and `-` and we only want to remove the leading. Here is some test cases for this: 
```rs
assert_eq!(to_valid_pkg_name("-2-123tau2ri-app-2"), "tau2ri-app-2");
assert_eq!(to_valid_pkg_name("1-2-3tau2ri-app2-"), "tau2ri-app2-");
```

Reply:
nvm, looks like this is correct, I should read the `skip_while` documentation instead of guessing how it works by its name.